OneCommunityGlobal/HGNRest
# Diego finished adding tests to postUserProfile

# Description
Adds missing unit tests to postUserProfile method in userProfileController

## Related PRS:
N/A

## Main changes explained:
- Refactors postUserProfile method and adds unit tests for positive and negative cases

## How to test:
1. check into current branch
2. do `npm install`
3. if working on MacOS or Linux run: `npm run prepare`
4. run: `npm run test /src/controllers/userProfileController.spec.js`
5. all tests should pass

## Screenshots or videos of changes:
https://github.com/OneCommunityGlobal/HGNRest/assets/26840798/48c44a68-7025-403f-8f8f-2f59019e1f82

## Note:
This unit tests covers positive and negative cases for postUserProfile method userProfileController
